backend/src/BuiltinCliHost/Libs/Cli.fs

```diff
@@ -152,10 +152,24 @@ let fns : List<BuiltInFn> =
                 | Ok dval -> return dval
                 | Error(rte, _cs) ->
                   let! rteString = Exe.runtimeErrorToString exeState rte
-                  return
-                    Exception.raiseInternal
-                      "Error executing pm function"
-                      [ "rte", rteString ]
+                  match rteString with
+                  | Ok(DString rte) ->
+                    return
+                      Exception.raiseInternal
+                        "Error executing pm function"
+                        [ "rte", rte ]
+                  | Ok dval ->
```

Comment:
do we need both `Ok(DString rte)` and `Ok dval`? or should we just keep the `Ok dval`?